pkg/workflows/wasm/host/wasm.go

```diff
@@ -41,5 +42,33 @@ func GetWorkflowSpec(ctx context.Context, modCfg *ModuleConfig, binary []byte, c
 
 	m.Close()
 
-	return wasmpb.ProtoToWorkflowSpec(sr)
+	return legacywasmpb.ProtoToWorkflowSpec(sr)
+}
+
+func GetTriggersSpec(ctx context.Context, modCfg *ModuleConfig, binary []byte, config []byte) (*sdkpb.TriggerSubscriptionRequest, error) {
```

Comment:
This is only called from a test. Do we need it? Shouldn't the test just call Execute() to get triggers...? Why expose this "backdoor"?

Reply:
Up to you. I thought it would be helpful for the engine, since it returns the correct types. It wasn't meant to be a backdoor. It calls Execute and decodes the response for you.